envoy-control-runner/src/main/resources/application.yaml

```diff
@@ -17,4 +17,9 @@ management:
     metrics.enabled: true
     prometheus.enabled: true
   endpoints.web.exposure.include: "*"
-  metrics.export.prometheus.enabled: true
\ No newline at end of file
+  metrics.export.prometheus.enabled: true
+
+spring:
+  jackson:
+    deserialization:
+      fail-on-unknown-properties: false
```

Comment:
Why is this needed?

Reply:
Tested something and didn't notice. I ll remove it